File: Later Work/C&Unix/fowlerblab2/syscalls.c
//CSCD340
//Lab2
//Brandon Fowler

#include <stdlib.h>
#include <stdio.h>

int main(){
	FILE * fin = fopen("test.txt","r");
	char temp[100];
	fgets(temp,100,fin);
	while(!feof(fin)){
		printf("%s",temp);
		fgets(temp,100,fin);
	}
	fclose(fin);
	return 0;
}

File: Later Work/C&Unix/fowlerblab2/SysCalls.java
//CSCD340
//Lab2
//Brandon Fowler

import java.util.Scanner;
import java.io.*;

public class SysCalls{
	public static void main(String[] args)throws FileNotFoundException{
		Scanner fin = new Scanner(new File("test.txt"));
		while(fin.hasNext()){
			String str = fin.nextLine();
			System.out.println(str);
		}
		fin.close();
	}
}
